recipes/qonnx/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "qonnx" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/qonnx-{{ version }}.tar.gz
+  sha256: dcb87eb74bec8a84d98e1746e4896c0184368f7cf377d0f5ee11179958ab84d3
+
+build:
+  noarch: generic
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  entry_points:
+    - qonnx-cleanup = qonnx.util.cleanup:main
+    - qonnx-exec = qonnx.util.exec_qonnx:main
+    - qonnx-to-channels-last = qonnx.util.to_channels_last:main
+    - qonnx-inference-cost = qonnx.util.inference_cost:main
+    - qonnx-convert = qonnx.util.convert:main
+    - qonnx-range-analysis = qonnx.util.range_analysis:main
+    - qonnx-download-model = qonnx.util.test:qonnx_download_model
+
+requirements:
+  host:
+    - python >=3.8
+    - setuptools >=46.1.0
+    - setuptools-scm >=5
+    - wheel
+    - pip
+  run:
+    - python >=3.8
+    - attrs >=22.2.0
+    - clize >=5.0.1
+    - bitstring >=3.1.7
+    - numpy >=1.24.1
+    - onnx >=1.13.0
+    - onnxruntime >=1.15.0
+    - sigtools >=4.0.1
+    - toposort >=1.7.0
+
+test:
+  imports:
+    - qonnx
+  commands:
+    # the package has a very tight protobuf pin
+    # but it protobuf is not used.
```

Comment:
Also needs https://github.com/conda-forge/clize-feedstock/pull/9 and https://github.com/conda-forge/sigtools-feedstock/pull/16 before pip check can be enabled.